workflows/charts/tgi/values.yaml

```diff
@@ -0,0 +1,64 @@
+# Copyright (c) 2024 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# -- Name of the serving service
+nameOverride: ""
+# -- Full qualified Domain Name
+fullnameOverride: ""
+deploy:
+  # -- ConfigMap of Environment Variables
+  configMapName: intel-proxy-config
+  # -- Intel TGI Image
+  image: ghcr.io/huggingface/text-generation-inference:latest-intel
+  # -- Model to be loaded
+  model: HuggingFaceTB/SmolLM-135M
+  # -- Enable Quantization (ex: bitsandbytes-nf4)
+  quantize: ""
+  # -- Number of pods
+  replicaCount: 1
+  # -- Resource configuration
+  resources:
+    limits:
+      cpu: 4000m
+      # -- Intel GPU Device Configuration
+      gpu.intel.com/i915: 1
+      # habana.ai/gaudi: 1
+      # memory: 409Gi
+      # hugepages-2Mi: 95000Mi
+    requests:
+      cpu: 1000m
+      memory: "1Gi"
+secret:
+  # -- Base64 Encoded Huggingface Hub API Token
+  encodedToken: nil
```

Comment:
Is there a way to check and let user know if the token is not set? Right now it does fail if the token isn't given but the error message isn't easy to understand.

For your reference here's the error message you get:

`* Secret in version "v1" cannot be handled as a Secret: illegal base64 data at input byte 0`

Reply:
Helm's error message here is that check. Helm does not support types or type checking without a very novel solution.

Reply:
The error message you see here is from kubernetes rejecting the manifest.

Reply:
Helm's [required function](https://helm.sh/docs/howto/charts_tips_and_tricks/#using-the-required-function) might help here?

Reply:
I added the requirement, thanks for the suggestion!